Source/AwsCommonRuntimeKit/http/HTTPVersion.swift

```diff
@@ -0,0 +1,26 @@
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0.
+import AwsCHttp
+
+public enum HTTPVersion {
+    case unknown // Invalid version
+    case version_1_0
```

Comment:
We can remove this as SDK do not support version 1.0

Reply:
mark it as deprecated with message
```suggestion
    @available(*, deprecated, message: "Not supported, use version_1_1")
    case version_1_0
```

Reply:
thinking more or can we just not have it, this never existed before right?